src/test/java/org/apache/commons/io/output/DeferredFileOutputStreamTest.java

```diff
@@ -365,4 +370,55 @@ private void verifyResultFile(final File testFile) {
             fail("Unexpected IOException");
         }
     }
+
+    /**
+     * Tests the case where the amount of data falls below the threshold, and is therefore confined to memory.
+     * Testing the getInputStream() method.
+     */
+    @ParameterizedTest(name = "initialBufferSize = {0}")
+    @MethodSource("data")
+    public void testBelowThresholdGetInputStream(final int initialBufferSize) throws IOException {
+        final DeferredFileOutputStream dfos = new DeferredFileOutputStream(testBytes.length + 42, initialBufferSize,
```

Comment:
Use try-with-resources.

Reply:
Done.

Reply:
Won't work here since the `DeferredFileOutputStream` needs to be closed while it is still in scope in order to execute `toInputStream()`